packages/mdc-select/component.ts

```diff
@@ -132,30 +130,8 @@ export class MDCSelect extends MDCComponent<MDCSelectFoundation> {
     };
     this.handleKeydown_ = (evt) => this.foundation_.handleKeydown(evt);
     this.handleMenuSelected_ = (evtData) => this.selectedIndex = evtData.detail.index;
```

Comment:
Delegate this to foundation? Also, rename this to `handleMenuItemAction` to be consistent with menu component.